.github/workflows/ci.yml

```diff
@@ -90,3 +90,34 @@ jobs:
         run: yarn workspace basic-app test:mocha
       - name: Custom App
         run: yarn workspace custom-fastboot-app test:mocha
+
+  test-legacy-mocha:
+    name: Legacy Mocha Tests - ${{ matrix.node-version }}
```

Comment:
I wonder what is legacy about these

Reply:
I agree that we probably shouldn't call them legacy 🤔  but I wouldn't think it's a blocker